### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by jordanafung**

### CODE REVIEW

**Comment by jordanafung** [comment]
I am unsure how changing the title will affect the LHS @neflyte 
Perhaps you can show me what else I need to do in these cases?

File: site/content/contribute/good-decisions/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Making good decisions"
-heading: "Making good decisions"
+title: "Community Expectations"
```
Comment (cwarnermm) [inline site/content/contribute/good-decisions/_index.md:2]:
```suggestion
title: "Community expectations"
```

File: site/content/contribute/good-decisions/_index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+heading: "Community Expectations"
```
Comment (cwarnermm) [inline site/content/contribute/good-decisions/_index.md:3]:
```suggestion
heading: "Community expectations"
```

**Review by cwarnermm — COMMENTED** [review]
Really like your clarifications and updates to this content, @jordanafung! Thank you! I've provided inline feedback to update titles from title case to sentence case.

**Comment by cwarnermm** [comment]
@jordanafung - I've added the **Preview-environment** label to this PR to generate previews with new commits. Links appear to be working as expected in the latest preview.

**Comment by jordanafung** [comment]
Thanks @cwarnermm !
I've commited your changes, and added a new commit for adding a note about the CoC and community expectations to the Contributor expectations page.

**Review by cwarnermm — APPROVED** [review]

**Review by neflyte — APPROVED** [review]

File: site/content/contribute/why-contribute/_index.md [hunk 3]
```diff
@@ -109,7 +110,7 @@ Thank you for your enthusiasm! You can act on feature ideas in a few ways:
 
 We love developers who are passionate about open-source! 
 
-If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue, and follow the process outlined in [You've found a bug](#youve-found-a-bug) of this guide.
+If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue, and follow the process outlined in [You've found a bug](#youve-found-a-bug) of this guide. You can find a list of the Mattermost repos on the [Contributor expectations page]({{< ref "/contribute/expectations/#before-contributing" >}})of this guide.
```
Comment (jordanafung) [inline site/content/contribute/why-contribute/_index.md:113]:
```suggestion
If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue, and follow the process outlined in [You've found a bug](#youve-found-a-bug) of this guide. You can find a list of the Mattermost repos on the [Contributor expectations page]({{< ref "/contribute/expectations/#before-contributing" >}}) of this guide.
```

**Review by cwarnermm — APPROVED** [review]

**Review by azigler — APPROVED** [review]
Really great work @jordanafung! I'm approving the PR because the content is in good shape to be consumed, but I left a few comments for you to consider as you iterate.

File: site/content/contribute/expectations/_index.md [hunk 4]
```diff
@@ -5,6 +5,7 @@ weight: 3
 ---
 
 To contribute to Mattermost, you must sign the [Contributor License Agreement](https://mattermost.com/mattermost-contributor-agreement/). Doing so adds you to our list of [Mattermost Approved Contributors](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). 
+Please also read our [community expectations]({{< ref "/contribute/good-decisions/" >}}) and note that we all abide by the [Mattermost Code of Conduct (CoC)](https://handbook.mattermost.com/contributors/contributors/guidelines/contribution-guidelines), and by joining our contributor community, you agree to abide by it as well.
 
 **Tip:** Love swag? If you choose to provide us with your mailing address in the signed agreement, you'll receive a [Limited Edition Mattermost Mug](https://forum.mattermost.com/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged.
```
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/expectations/_index.md:10]: On this page https://docs.mattermost.com/install/troubleshooting.html#mattermost-server-logs there are Note boxes.
They aren't as pretty - but they can help break up the text a bit?
@azigler

File: site/content/contribute/good-decisions/_index.md [hunk 5]
```diff
@@ -9,28 +9,30 @@ To be an open, safe, and welcoming community, we strive to be inclusive, collabo
 
 Learn more about our {{<newtabref title="company values" href="https://handbook.mattermost.com/company/about-mattermost#leadership-principles">}}, and how to become a successful member of the Mattermost community by learning and following our standard operational guidelines below. Please read these sections below carefully and let us know if you have any questions or concerns.
```
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/good-decisions/_index.md:10]: I saw that as well and it's due to the padding of the link, I think

File: site/content/contribute/good-decisions/_index.md [hunk 6]
```diff
@@ -9,28 +9,30 @@ To be an open, safe, and welcoming community, we strive to be inclusive, collabo
...
-## We're collaborative
+## We are collaborative
   - We ask questions and consult others.
   - We work together and help each other.
   - We aim for clarity.
 
-## We're considerate
+## We are considerate
   - We have patience with each other.
   - No one has all the answers, nor are they expected to.
```
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/good-decisions/_index.md:29]:
```suggestion
  - We understand that no one has all the answers, nor are they expected to.
```

File: site/content/contribute/good-decisions/_index.md [hunk 7]
```diff
@@ -9,28 +9,30 @@ To be an open, safe, and welcoming community, we strive to be inclusive, collabo
...
 
-## We're respectful
+## We are respectful
   - We offer thanks and we’re grateful.
   - We may occasionally disagree, but we resolve these disagreements in respectful ways, take breaks if things get heated, reassess, and consult others where appropriate.
   - We aim to be self-aware and we take responsibility for our impact through our words and actions.
   - We understand and acknowledge that {{<newtabref title="intent doesn’t equal impact" href="https://www.betterup.com/blog/intent-vs-impact">}}. We can have the best of intentions, but still cause negative impact to others by our words and actions. This can happen to all of us, so we practice openness and grace.
 
+### Attribution
+This document was heavily inspired on and adapted from the work of the [Drupal Community and its Code of Conduct](https://www.drupal.org/dcoc) and its [Values and Principles](https://www.drupal.org/about/values-and-principles) documents.
```
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/good-decisions/_index.md:38]:
```suggestion
This document was heavily inspired by and adapted from the work of the [Drupal Community and its Code of Conduct](https://www.drupal.org/dcoc) and its [Values and Principles](https://www.drupal.org/about/values-and-principles) documents.
```

File: site/content/contribute/expectations/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/expectations/_index.md:10]: Oh! And these are from the docs.mm site, which is I think a different repo and may have different thingamabobs to play with.
Maybe we should save this comment for a future iteration when we figure out our options @cwarnermm ?

[the remark this answers is not in the document]
Comment (azigler) [reply site/content/contribute/expectations/_index.md:10]: Also, line 15 below here, "You can create an app integration, which connects to Mattermost’s functionality that's hosted elsewhere."

Maybe that could be rephrased as: "You can create app integrations for Mattermost to provide additional functionality and host them anywhere."

File: site/content/contribute/good-decisions/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (cwarnermm) [reply site/content/contribute/good-decisions/_index.md:10]: The extra empty space following links is something @neflyte has fixed in another PR. Once that PR is merged, that fix will apply site-wide.

File: site/content/contribute/expectations/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cwarnermm) [reply site/content/contribute/expectations/_index.md:10]: @neflyte has a card on his board to ensure all tip & note blocks are formatted using a consistent approach that's easy to see/read at a glance. He can apply this update site-wide.

File: site/content/contribute/good-decisions/_index.md [hunk 8]
```diff
@@ -9,28 +9,30 @@ To be an open, safe, and welcoming community, we strive to be inclusive, collabo
 
 Learn more about our {{<newtabref title="company values" href="https://handbook.mattermost.com/company/about-mattermost#leadership-principles">}}, and how to become a successful member of the Mattermost community by learning and following our standard operational guidelines below. Please read these sections below carefully and let us know if you have any questions or concerns.
 
-## We're inclusive
+## We are inclusive
 
 We welcome all people, **but not all behavior**.
```
[the remark this answers is not in the document]
Comment (jordanafung) [reply site/content/contribute/good-decisions/_index.md:14]: Adding this to our working draft doc, to address in a future iteration
Thanks Andrew!

**Pull request merged**